context/src/main/java/io/micronaut/scheduling/executor/ExecutorType.java

```diff
@@ -45,7 +45,7 @@ public enum ExecutorType {
     WORK_STEALING,
 
     /**
-     * @see java.util.concurrent.Executors#newThreadPerTaskExecutor()
+     * @see io.micronaut.scheduling.LoomSupport#newThreadPerTaskExecutor(java.util.concurrent.ThreadFactory)
```

Comment:
@yawkat is this change in the javadoc correct?

Reply:
no. the old one is correct. i assume it triggers a lint failure because it points to a method that is not available on java 17, but that's fine imo.

Reply:
@yawkat hm... yes, it's true. But why it's Ok if we use java 17 and java 17 haven't this method in JDK ?